python/lsst/pipe/tasks/makeCoaddTempExp.py

```diff
@@ -164,8 +183,22 @@ def createTempExp(self, calexpRefList, skyInfo, visitId=0):
                 calExpRef = calExpRef.butlerSubset.butler.dataRef("calexp", dataId=calExpRef.dataId,
                                                                   tract=skyInfo.tractInfo.getId())
                 calExp = self.getCalExp(calExpRef, bgSubtracted=self.config.bgSubtracted)
-                exposure = self.warpAndPsfMatch.run(calExp, modelPsf=modelPsf, wcs=skyInfo.wcs,
-                                                    maxBBox=skyInfo.bbox).exposure
+                covName = 'covar'
+                for key, val in calExpRef.dataId.items():
+                    covName += '_%s_%s'%(key, val)
```

Comment:
I was not sure if writing the covariance matrix for an individual exposure made sense but thought I'd leave the name code to do so in there anyway. I was planning on only writing out the per-visit covariance matrix.